docs/architecture/README.md

```
@@ -1,3 +1,13 @@
-# Docs - Architecture
+# Architecture Documentation
 
-This directory contains resources for docs/architecture.
+## Overview
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)